openmp/libomptarget/test/mapping/auto_zero_copy.cpp

```diff
@@ -0,0 +1,43 @@
+
+// RUN: %libomptarget-compilexx-generic
+// RUN: env HSA_XNACK=1 LIBOMPTARGET_DEBUG=1 %libomptarget-run-generic 2>&1 \
+// RUN: | %fcheck-generic -check-prefix=DEBUG -check-prefix=CHECK
+// REQUIRES: libomptarget-debug
```

Comment:
I wonder if we should prefer to use `LIBOMPTARGET_INFO` with the appropriate level to check for data copies. This is mostly motivated as we run buildbots with pure release builds w/o support for `LIBOMPTARGET_DEBUG`.

Reply:
I checked LIBOMPTARGET_INFO and it does not print anything related to this. If you think it makes sense, I can add an info print so we can check that the mapping of the arrays is done through auto zero-copy.

Reply:
INFO should print if there is a data copy, no? So the absence of that is what we could test for. In that case I'd like a negative test too though.

Reply:
I added an info message so I can check that and remove libomptarget-debug.